spec/index.html

```diff
@@ -10373,7 +10373,7 @@ <h3>Evaluation Semantics</h3>
                 where <var>Ω</var> is the multiset of solution mappings produced by the following algorithm:
                 <pre class="code nohighlight" style="font-family: sans-serif;">
 <var>Ω</var> := the empty multiset
-foreach <a data-cite="RDF12-CONCEPTS#dfn-graph-name">graph name</a> <var>gn</var> in <var>D</var>
+foreach <a data-cite="RDF12-CONCEPTS#dfn-graph-name">graph name</a> <var>gn</var> in <var>D</var> (recall that a graph name may be an IRI or a blank node)
```

Comment:
`foreach` is found in various programming languages. `for each` is the English phrase.
```suggestion
for each <a data-cite="RDF12-CONCEPTS#dfn-graph-name">graph name</a> <var>gn</var> in <var>D</var> (recall that a graph name may be an IRI or a blank node)
```